core/src/main/scala/org/apache/spark/Accumulable.scala

```diff
@@ -68,7 +68,9 @@ class Accumulable[R, T] private (
   }
 
   private[spark] def this(initialValue: R, param: AccumulableParam[R, T], name: Option[String]) = {
+    // scalastyle:off
     this(initialValue, param, name, false /* countFailedValues */)
+    // scalastyle:on
```

Comment:
I believe we could enable and disable via `scalastyle:on customid`  rather than switching off whole style rules here IIRC.